RangeTreeTests/NonStaticComparerTests.cs

```diff
@@ -1,109 +0,0 @@
-﻿namespace RangeTreeTests
```

Comment:
Is there a correspondency between the old tests and the new ones? Did you just trash the old tests and write new ones?